src/ucs/config/parser.c

```diff
@@ -2027,6 +2045,22 @@ ucs_config_parser_print_field(FILE *stream, const void *opts, const char *env_pr
     ucs_assert(!ucs_list_is_empty(prefix_list));
     head = ucs_list_head(prefix_list, ucs_config_parser_prefix_t, list);
 
+    if ( snprintf(name_buf, sizeof(name_buf), "%s%s%s", env_prefix, head->prefix, name) >= (int)(sizeof(name_buf))) {
```

Comment:
comde format - pls check clang-format output

Reply:
could use ucs_snprintf_safe without additional overflow checks

Reply:
> Are you sure we want to silently ignore when the buffer is not large enough, in which case the filter is not applied as expected? I'd want to draw attention to it so that we can increase the buffer size.

There are many places we build the var name this way, so IMO not worth checking it only when printing